Makefile

```diff
@@ -37,6 +37,9 @@ STRESSFLAGS  :=
 DUPLFLAGS    := -t 100
 
 ifeq ($(STATIC),1)
+# The Go resolver should work well enough for now; better than musl, anyway.
```

Comment:
- do you want to mention that despite netgo being the default, it doesn't seem to "take" when using cgo?
- could save a deref and google search by pointing to https://github.com/gliderlabs/docker-alpine/issues/8 which seems to be the most active discussion of this bug